### ISSUE TITLE
User Story from OS (24-02-2020) #3

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutSPACEs#346]
### I would like to...

be able to share my experiences without guilt or shame

### Because...

it's easier to be honest 

### Acceptance criteria

There is a good moderation process in place. 
The platform is a welcoming environment - for this we need to collect feedback on how autistic people find the experience

## Notes

* Associated pull request: 
* Current status:

---

**Comment by GeorgiaHCA** [comment]
This is part of the platform's MVP and should be integral
